tests/packageJson.test.js

```
@@ -0,0 +1,473 @@
+  console.log('Starting PackageJson test suite');
+});
+
+afterAll(() => {
+  console.log('Completed PackageJson test suite');
+});
+
+// Export test utilities for potential use by other test files
+module.exports = {
+  validPackageJson,
+  invalidPackageJson
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)